### PR TITLE
Update Council page with latest changes

### LINKED ISSUE #186: Update council page
We need to update the council page with this information: https://www.mautic.org/mautic-council.

### PR DESCRIPTION
**Pull request by domidc**
New names in the council added

Fixes #186

### CODE REVIEW

File: pages/04.community-leadership/03.community-council/docs.md [hunk 1]
```diff
@@ -20,11 +20,10 @@ The Community Council discusses topics that impact the community as a whole, or
 
 | Name      | Role     | Email address |
 |-----------|--------------------|-----------------------|
-| Ruth Cheesley | Mautic Project Lead at Acquia | ruth.cheesley@mautic.org            | 
-| Dries Buytaert | Acquia Co-founder and CTO | dries@acquia.com            | 
-| Gábor Hojtsy | Community Specialist | gabor.hojtsy@acquia.com         | 
+| Andy Towne | Senior customer value manager at Acquia | https://linkedin.com/in/andytowne            |
```
Comment (RCheesley) [inline pages/04.community-leadership/03.community-council/docs.md:23]:
```suggestion
| Andy Towne | Senior Customer Value Manager at Acquia | andy.towne@mautic.org           | 
```

File: pages/04.community-leadership/03.community-council/docs.md [hunk 2]
```diff
@@ -20,11 +20,10 @@ The Community Council discusses topics that impact the community as a whole, or
 
 | Name      | Role     | Email address |
 |-----------|--------------------|-----------------------|
-| Ruth Cheesley | Mautic Project Lead at Acquia | ruth.cheesley@mautic.org            | 
-| Dries Buytaert | Acquia Co-founder and CTO | dries@acquia.com            | 
-| Gábor Hojtsy | Community Specialist | gabor.hojtsy@acquia.com         | 
+| Andy Towne | Senior customer value manager at Acquia | https://linkedin.com/in/andytowne            | 
+| Dominique De Cooman | Dropsolid Co-founder and co-CEO | dominique@dropsolid.com            |
```
Comment (RCheesley) [inline pages/04.community-leadership/03.community-council/docs.md:24]:
```suggestion
| Dominique De Cooman | Dropsolid Co-founder and co-CEO | dominique.decooman@mautic.org           | 
```

File: pages/04.community-leadership/03.community-council/docs.md [hunk 3]
```diff
@@ -20,11 +20,10 @@ The Community Council discusses topics that impact the community as a whole, or
 
 | Name      | Role     | Email address |
 |-----------|--------------------|-----------------------|
-| Ruth Cheesley | Mautic Project Lead at Acquia | ruth.cheesley@mautic.org            | 
-| Dries Buytaert | Acquia Co-founder and CTO | dries@acquia.com            | 
-| Gábor Hojtsy | Community Specialist | gabor.hojtsy@acquia.com         | 
+| Andy Towne | Senior customer value manager at Acquia | https://linkedin.com/in/andytowne            | 
+| Dominique De Cooman | Dropsolid Co-founder and co-CEO | dominique@dropsolid.com            | 
+| Ekke Guembel | CEO at leuchtfeuer digital marketing | ekke.guembel@mautic.org        | 
 | John Linhart | Technical Community Lead at Acquia | jan.linhart@acquia.com           |
```
Comment (RCheesley) [inline pages/04.community-leadership/03.community-council/docs.md:26]:
```suggestion
| John Linhart | Principal Software Engineer at Acquia | john.linhart@mautic.org           |
```

File: pages/04.community-leadership/03.community-council/docs.md [hunk 4]
```diff
@@ -20,11 +20,10 @@ The Community Council discusses topics that impact the community as a whole, or
...
-| Ekkehard Gümbel  | Community Representative | ekke.guembel@mautic.org            | 
-| Oluwatobi Owolabi  | Community Representative | oluwatobi.owolabi@mautic.org            | 
-| Norman Pracht  | Community Representative | 	norman.pracht@mautic.org            | 
-| Favour Kelvin | Community Representative | favour.kelvin@mautic.org            | 
+| Favour Chibueze  | Community Representative | favour.kelvin@mautic.org             |
```
Comment (RCheesley) [inline pages/04.community-leadership/03.community-council/docs.md:27]:
```suggestion
| Favour Chibueze  | Front End Engineer at Eden Life Inc. | favour.chibueze@mautic.org             | 
```

File: pages/04.community-leadership/03.community-council/docs.md [hunk 5]
```diff
@@ -20,11 +20,10 @@ The Community Council discusses topics that impact the community as a whole, or
...
+| Andy Towne | Senior customer value manager at Acquia | https://linkedin.com/in/andytowne            | 
+| Dominique De Cooman | Dropsolid Co-founder and co-CEO | dominique@dropsolid.com            | 
+| Ekke Guembel | CEO at leuchtfeuer digital marketing | ekke.guembel@mautic.org        | 
 | John Linhart | Technical Community Lead at Acquia | jan.linhart@acquia.com           |
-| Ekkehard Gümbel  | Community Representative | ekke.guembel@mautic.org            | 
-| Oluwatobi Owolabi  | Community Representative | oluwatobi.owolabi@mautic.org            | 
-| Norman Pracht  | Community Representative | 	norman.pracht@mautic.org            | 
-| Favour Kelvin | Community Representative | favour.kelvin@mautic.org            | 
+| Favour Chibueze  | Community Representative | favour.kelvin@mautic.org             | 
+| Norman Pracht  | CEO Webmecanik | 	norman.pracht@mautic.org            |
```
Comment (RCheesley) [inline pages/04.community-leadership/03.community-council/docs.md:28]:
```suggestion
| Norman Pracht  | Associate General Manager, Webmecanik | 	norman.pracht@mautic.org            | 
```

File: pages/04.community-leadership/03.community-council/docs.md [hunk 6]
```diff
@@ -20,11 +20,10 @@ The Community Council discusses topics that impact the community as a whole, or
...
+| Dominique De Cooman | Dropsolid Co-founder and co-CEO | dominique@dropsolid.com            | 
+| Ekke Guembel | CEO at leuchtfeuer digital marketing | ekke.guembel@mautic.org        | 
 | John Linhart | Technical Community Lead at Acquia | jan.linhart@acquia.com           |
-| Ekkehard Gümbel  | Community Representative | ekke.guembel@mautic.org            | 
-| Oluwatobi Owolabi  | Community Representative | oluwatobi.owolabi@mautic.org            | 
-| Norman Pracht  | Community Representative | 	norman.pracht@mautic.org            | 
-| Favour Kelvin | Community Representative | favour.kelvin@mautic.org            | 
+| Favour Chibueze  | Community Representative | favour.kelvin@mautic.org             | 
+| Norman Pracht  | CEO Webmecanik | 	norman.pracht@mautic.org            | 
+| Prateek Jain | Director of digital experience services at Axelerant | prateek@axelerant.com            |
```
Comment (RCheesley) [inline pages/04.community-leadership/03.community-council/docs.md:29]:
```suggestion
| Prateek Jain | Director of Digital Experience Services at Axelerant | prateek.jain@mautic.org           | 
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
I have updated the list to include Mautic emails rather than professional emails, and also to correct a few inaccuracies with job titles.

**Review by domidc — COMMENTED** [review]
Reviewed

**Review by npracht — APPROVED** [review]

**Comment by RCheesley** [comment]
@all-contributors please add @domidc for docs

**Pull request merged**